roqoqo-qryd/tests/integration/api_devices.rs

```diff
@@ -578,6 +629,10 @@ fn test_to_generic_device_triangular() {
 fn test_phi_theta_relation() {
     let triangular = QrydEmuTriangularDevice::new(Some(0), None, None);
     let square = QrydEmuSquareDevice::new(Some(0), None, None);
+    let triangular_f =
+        QrydEmuTriangularDevice::new(Some(0), Some("2.13".to_string()), Some("2.25".to_string()));
+    let square_f =
+        QrydEmuSquareDevice::new(Some(0), Some("2.13".to_string()), Some("2.25".to_string()));
```

Comment:
The unittest also need to be adapted since we do not want to have the overwrite for ControlledPhase